api/collections/serializers.py

```diff
@@ -94,3 +94,76 @@ def get_absolute_url(self, obj):
                 'node_link_id': obj._id
             }
         )
+
+
+class SingleLinkedNode(JSONAPIRelationshipsSerializer):
+    id = ser.CharField(source='node._id', required=False, allow_null=True)
```

Comment:
Does this need to be node specific? From the serializer level, aren't all relationships the same? You'd still have to do custom update etc, but the fields are all going to be an item id and a type, right? It's only a few lines of code, so if we want to boilerplate it, it's not a big deal. I know most of the work is in the update, delete, and create methods.